analyzer_test.go

```diff
@@ -9,34 +9,22 @@ import (
 	"os"
 	"path/filepath"
 	"testing"
+
+	"github.com/golang/dep/test"
 )
 
 func TestDeriveManifestAndLock(t *testing.T) {
-	dir, err := ioutil.TempDir("", "dep")
-	if err != nil {
-		t.Fatal(err)
-	}
-	defer os.RemoveAll(dir)
+	h := test.NewHelper(t)
+	defer h.Cleanup()
 
-	contents := `{
-    "dependencies": {
-        "github.com/pkg/errors": {
-            "version": ">=0.8.0, <1.0.0"
-        },
-        "github.com/sdboyer/gps": {
-            "version": ">=0.12.0, <1.0.0"
-        }
-    }
-}
-`
-
-	if err := ioutil.WriteFile(filepath.Join(dir, ManifestName), []byte(contents), 0644); err != nil {
-		t.Fatal(err)
-	}
+	h.TempDir("dep")
+	golden := "analyzer/manifest.json"
+	contents := h.GetTestFileString(golden)
```

Comment:
Oh, good call. IMO, we should try to start setting this bar and sticking with it. Sorry, @tro3, I realize this is dragging on. - do you mind making another change?